src/kernels/Conv_Winograd_v30_3_1_metadata.inc

```diff
@@ -70,8 +70,8 @@ amdhsa.kernels:
     - { .size: 4, .offset:  80, .value_kind: by_value, .value_type: i32, .name: out_h }
     - { .size: 4, .offset:  84, .value_kind: by_value, .value_type: i32, .name: out_w }
     - { .size: 8, .offset:  88, .value_kind: global_buffer, .value_type: f32, .name: bias_addr,    .address_space: global, .is_const: true }
-    - { .size: 4, .offset:  96, .value_kind: by_value, .value_type: f32, .name: RELU_alpha }
-    - { .size: 4, .offset: 100, .value_kind: by_value, .value_type: i32, .name: reserved2 }
+    - { .size: 4, .offset:  96, .value_kind: by_value, .value_type: f32, .name: alpha }
```

Comment:
I would like to ask somebody to doublecheck this file.

Reply:
LGTM